mkdocs.yml

```diff
@@ -2,11 +2,12 @@ site_name: alps-uenv
 nav:
   - Home: index.md
   - 'uenv':
-    - 'gromacs': uenv-gromacs.md
-    - 'quantum-espresso': uenv-qe.md
-    - 'cp2k': uenv-cp2k.md
-    - 'linaro-forge': uenv-linaro-forge.md
-  - 'packaging':
+    - 'CP2K': uenv-cp2k.md
```

Comment:
the names matched the names of the uenv on the systems, i.e the name that is displayed with `uenv image find` and used for `uenv start`, `srun --uenv=` etc.

How about having both the full name and the uenv name in the header (e.g. 'Linaro Forge: `linaro-forge`')?

Reply:
I see. I thought it was a bit ugly to have the `uenv` name in the menu, but what bothered me more was the ordering. I can revert the name if preferred, but your suggestion sounds good too.

Reply:
Let's use my suggestion, which will hopefully make the page more searchable.